02/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM mysql:8.0
+
+USER root
+
+# busyboxイメージを使って、必要なツールをコピー
+COPY --from=busybox:1.31.1-uclibc /bin/busybox /bin/busybox
+
+# busyboxを使ってunzipをシンボリックリンクで使用可能にする
+RUN ln -s /bin/busybox /bin/unzip
+
+WORKDIR /app
+
+RUN curl -L -o repo.zip https://github.com/datacharmer/test_db/archive/refs/heads/master.zip && \
+    busybox unzip repo.zip && \
+    rm repo.zip
```

「MySQLコンテナ内でダウンロードできるようにする」よりは「ダウンロードしたあと、MySQLコンテナ内に持っていく」方が良いかも？です

```suggestion
FROM busybox as helper
WORKDIR /workspace
RUN curl -L -o repo.zip https://github.com/datacharmer/test_db/archive/refs/heads/master.zip
RUN unzip repo.zip

FROM mysql:8.0
COPY --from=helper /workspace/repo /docker-entrypoint-initdb.d
```

---

完全にこれだ！！ありがとうございます
かりばやしさんが言ってたのもこれですかね

---

言葉足らずで申し訳ないです。。  
「ざっくりこんな感じ！」で雑に書いたので、動くかわからん（というか、動かない可能性大）です🙇‍♂️  
実際にコンテナ再作成したりして動作確認してほしいです🙇‍♂️

---

動かなかったので動かしました！